enterprise/app/filter/filter.tsx

```diff
@@ -456,6 +463,11 @@ export default class FilterComponent extends React.Component<FilterProps, State>
                 {compactDurationSec(Number(maximumDurationValue))}
               </span>
             )}
+            {dimensions.map((v) => (
+              <span className="advanced-badge" title={getDimensionName(v.dimension ?? undefined)}>
+                {getDimensionIcon(v.dimension ?? undefined)} {v.value}
+              </span>
```

Comment:
nit: would it make sense to do `v.dimension && <span ...>` instead of the `?? undefined` stuff?

Reply:
sure yeah, done